src/export/video_export.cpp

```diff
@@ -87,43 +87,61 @@ bool VideoExporter::isFFmpegAvailable()
     return !ffmpegPath.isEmpty();
 }
 
+QString VideoExporter::ffmpegCommandName()
+{
+#if Q_OS_WIN
+    return "ffmpeg.exe";
+#else
+    return "ffmpeg";
+#endif
+}
+
+QStringList VideoExporter::listFfmpegPossiblePaths()
+{
+#ifdef Q_OS_WIN
+    return {
+        "C:/Program Files/ffmpeg/bin/ffmpeg.exe",
```

Comment:
this line and the next are going to result in a ffmpegFile that will be "C:/Program Files/ffmpeg/bin/ffmpeg.exe/ffmpeg.exe" on Windows.

Reply:
Good catch, fixed in latest version.